Sources/Credential/Credential Base Classes/BaseOAuthSwiftCredential.swift

```diff
@@ -119,9 +61,9 @@ open class OAuthSwiftCredential: NSObject, NSCoding {
         self.oauthTokenSecret = (decoder.decodeObject(forKey: CodingKeys.oauthTokenSecret) as? String) ?? String()
         self.oauthVerifier = (decoder.decodeObject(forKey: CodingKeys.oauthVerifier) as? String) ?? String()
         self.oauthTokenExpiresAt = (decoder.decodeObject(forKey: CodingKeys.oauthTokenExpiresAt) as? Date)
-        self.version = Version(decoder.decodeInt32(forKey: CodingKeys.version))
+        self.version = (decoder.decodeObject(forKey: CodingKeys.version) as! OAuthSwiftVersion)
     }
```

Comment:
old stored credentials will not be readable (one way is to change the key and try to read the two)
force cast is not recommended, If you have swiftlint, there is an error